client/src/components/webPlayback.jsx

```diff
@@ -18,32 +18,21 @@ class WebPlayback extends React.Component {
     super(props);
 
     this.state = {
-      // is_paused is used to track the paused/play button
       is_paused: false,
-      // is_active is used to check if we should render the normal UI, or a loading/empty screen
       is_active: false,
-      // what is player used?
       player: undefined,
-      // is_playing_left_track is created 
-      // to help check if left/right song is playing,
-      // in order to run a specific song.
       is_playing_left_track: true,
-      // left and right side track are used to grab rendering information, such as album name and artist name.
       left_side_track: track,
       right_side_track: track,
-      // previous_track is there to help with left_side/right_side track.
-      previous_track: false,
-      // when sending final playlist to spotify api to create a playlist, spotify receives an array of spotify uris
       final_playlist: [],
+      gametracks: [],
```

Comment:
Used to keep track of 2 first song of the next set. This is used to make sure the user is choosing songs.